examples/anymcu-blink/platformio.ini

```diff
@@ -0,0 +1,13 @@
+; PlatformIO Project Configuration File

```

Comment:
Please add new examples to the CI list
https://github.com/platformio/platform-intel_mcs51/blob/develop/.github/workflows/examples.yml#L12